rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright © Aptos Foundation
+
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::parquet_signatures::Signature;
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::fungible_asset_models::v2_fungible_asset_utils::FeeStatement,
+    utils::util::{get_entry_function_from_user_request, parse_timestamp, standardize_address},
+};
+use allocative::Allocative;
+use anyhow::Result;
+use aptos_protos::{
+    transaction::v1::{
+        TransactionInfo, UserTransaction as UserTransactionPB, UserTransactionRequest,
+    },
+    util::timestamp::Timestamp,
+};
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(Allocative, Clone, Debug, Default, Deserialize, ParquetRecordWriter, Serialize)]
+pub struct UserTransaction {
+    pub txn_version: i64,
+    pub block_height: i64,
+    pub parent_signature_type: String,
+    pub sender: String,
+    pub sequence_number: i64,
+    pub max_gas_amount: u64,
+    pub expiration_timestamp_secs: u64,
+    pub gas_unit_price: u64,
+    pub gas_used: u64,
+    #[allocative(skip)]
+    pub timestamp: chrono::NaiveDateTime,
```

Comment:
can you reorder + rename columns to be

1. txn_version
2. block_height
3. block_timestamp (from timestamp)
4. epoch
5. sender
6. sequence_number
7. entry_function_id_str
8. expiration_timestamp_secs
9. parent_signature_type
10. gas_fee_payer_address
12. gas_used
13. gas_unit_price
14. max_gas_amount
15. storage_refund_amount -> storage_refund_octa
16. is_transaction_success
17. num_signatures

(not sure what the skip thing means)

Reply:
skip means skip right @yuunlimm ?

Reply:
yeah, we could use different way to measure the size, but i don't think it's a big deal

Reply:
reordered.

Reply:
hmm that's true, i redo the order in bigquery i think

one more rename: max_gas_amount -> max_gas_octa

Reply:
also does `gas_fee_payer_address` need standardization? (faa has correct length)

Reply:
address is already standardized.